model-optimizer/mo/graph/perm_inputs.py

```diff
@@ -171,6 +171,24 @@ def transpose(op_node: Node, port_info: str, input_port: int):
     op_node.in_port(input_port).get_connection().insert_node(transpose)
 
 
+def transpose_nchw_to_nhwc(op_node: Node, port_info: str, input_port: int):
```

Comment:
Also, as I understant the function is not adjusted to be called using new Graph API, namely, I can access data node but dataaccessor.